src/components/NavBar.js

```diff
@@ -0,0 +1,39 @@
+import { NavLink } from 'react-router-dom';
+import style from '../styles/NavBar.module.css';
+
+const NavBar = () => {
+  const styleActive = `${style.active} ${style.link}`;
+  const styleNoActive = `${style.noActive} ${style.link}`;
+  return (
+    <nav className={style.nav}>
+      <ul className={style.ul}>
+        <li>
+          <NavLink
+            to="/"
+            className={({ isActive }) => (isActive ? styleActive : styleNoActive)}
+          >
+            Home
+          </NavLink>
```

Comment:
- [ ] I like how you are using `NavLink` to apply active styles. Great job. :+1: